openvpn/_mapdata/init.sls

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+---
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import map as mapdata with context %}
+
+{%- do salt['log.debug']('### MAP.JINJA DUMP ###\n' ~ mapdata | yaml(False)) %}
+
+{%- set output_dir = '/temp' if grains.os_family == 'Windows' else '/tmp' %}
```

Comment:
This `/temp` on windows became a CI prerequisite, right?

Someone with a pristine windows will not have this directory.

Reply:
@baby-gnu I understand that we can do this "properly" but since these files are going to be managed by the `ssf-formula` in the long run, I don't want to worry about much more than our current situation.  We have two Windows platforms to consider and `/temp` works on both of them out of the box:

1. `Windows-2019Server` here in GitHub Actions.
1. `Windows-8.1` available for local testing via. Vagrant.

Since all of our testing is built around the containers/VMs that we ourselves provide, I believe we should avoid unnecessary complexity until the situation arises.  Besides, there aren't many circumstances where the `_mapdata` testing becomes important other than from a SaltStack Formulas org perspective (i.e. for reviewing PRs).

Another angle is that many of the formulas don't have Windows testing in the first place.  So I dislike introducing excessive complexity unnecessarily.  We want to make contributing as easy as possible.  I think this use of `/tmp` and `/temp` is simple enough to work with, while attaining the required objective.

Reply:
OK